reporter/src/main/kotlin/utils/SpdxDocumentModelMapper.kt

```diff
@@ -76,10 +78,21 @@ object SpdxDocumentModelMapper {
         ortResult.getPackages(omitExcluded = true).forEach { curatedPackage ->
             val pkg = curatedPackage.pkg
 
+            if (pkg.vcsProcessed.url.isNotBlank()) {
```

Comment:
Why do you check vcsProcessed while at the same time the PURL does not contain the VCS but the binary artifact URL?

Reply:
`pkg.vcsProcessed.url.isNotBlank()` was a copy-paste mistake fixed it now should have been `pkg.purl.isNotBlank()`